### PR TITLE
after first browser render, check sizes again (fixes #8)

### LINKED ISSUE #8: Size class isn't applied on init stage
Could you please tell me what I'm doing wrong or is it a well-known issue that you need to resize your browser to make classes be applied?

### PR DESCRIPTION
**Pull request by jokeyrhyme**
I've added a second call to `sizeComponent()` a few frames after `componentDidMount()`.

I tried `setTimeout(fn, 0)` and that was too early, and I also tried `setTimeout(fn, 1e3)` and it worked but the delay was far too visibly obvious.

Let me know if you'd like a squash, or if you'd like me to undo some of the other changes I snuck in (as separate commits), or if you have other comments.

Fixes #8

### CODE REVIEW

**Comment by joeybaker** [comment]
Thanks @jokeyrhyme! Can you take a look at why the tests are failing?

**Comment by jokeyrhyme** [comment]
Weird. It's passing the browser tests now in Travis CI, but is still coming up with a failure. I'll look into this later today perhaps.

**Comment by joeybaker** [comment]
Thanks @jokeyrhyme!

File: index.jsx [hunk 1]
```diff
@@ -110,18 +115,18 @@ static _componentMap = new Map()
     let matchedSize = ''
     let matchedWidth = smallestSize.width
 
+    // use Array#some() here because #forEach() has no early exit
```
Comment (joeybaker) [inline index.jsx]: Thanks for the clarification!

Comment (jokeyrhyme) [inline index.jsx]: Yeah, it took me a second to realise why that choice was made, and I figured it was worth spelling out. :)

**Comment by jokeyrhyme** [comment]
@joeybaker there we go. It was a Travis CI environment issue: not having an X service available for Firefox to use. Green!

File: .travis.yml [hunk 2]
```diff
@@ -13,3 +14,9 @@ sudo: false
 
 before_install:
  - "npm install -g npm@latest"
+
+# https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI
+before_script:
```
Comment (joeybaker) [inline .travis.yml]: Weird that this didn't break before. Thanks!

**Comment by joeybaker** [comment]
@jokeyrhyme Thanks! Released as v2.0.2, and added you as a collaborator. Just use `npm run release [patch|minor|major]` to publish to npm.

**Pull request merged**